news/views.py

```diff
@@ -526,6 +526,15 @@ def newsletters(request):
     })
 
 
+@require_safe
+@cache_page(30)
+def healthz(request):
+    """for use with healthchecks. wrapped with `cache_page` to test redis cache."""
```

Comment:
that'd be several hundred db hits per minute. I like healthchecks but that seems excessive. Would you like it better if I backed off the cache to just 10 seconds?